tap_exacttarget/dao.py

```diff
@@ -59,6 +59,12 @@ def get_catalog_keys(self):
     def parse_object(self, obj):
         return project(obj, self.get_catalog_keys())
 
+    # a function to write records with applying transformation
+    def write_records_with_transform(self, record, catalog, table): # pylint: disable=no-self-use
```

Comment:
@harshpatel4crest Can you please try resolving this pylint error instead of disabling it

Reply:
Done